core/arch/arm/kernel/link.mk

```diff
@@ -163,14 +163,15 @@ CORE_CC_VERSION = `$(CCcore) -v 2>&1 | grep "version " | sed 's/ *$$//'`
 define gen-version-o
 	$(call update-buildcount,$(link-out-dir)/.buildcount)
 	@$(cmd-echo-silent) '  GEN     $(link-out-dir)/version.o'
-	$(q)echo -e "const char core_v_str[] =" \
+	$(q)cd $(link-out-dir) && \
+		$(q)echo -e "const char core_v_str[] =" \
```

Comment:
You need to remove the `$(q)` here (that's what makes CI fail)

Reply:
Yep, done now.